extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -74,6 +91,8 @@ export const renderPageAttachmentUI = (win, pageEl, attachmentEl) => {
   const attachmentHref = attachmentEl.getAttribute('href');
   if (isPageAttachmentUiV2ExperimentOn(win) && !attachmentHref && openImgAttr) {
     return renderPageAttachmentUiWithImages(win, pageEl, attachmentEl);
+  } else if (isPageAttachmentUiV2ExperimentOn(win) && attachmentHref) {
+    return renderOutlinkPageAttachmentUI(win, pageEl, attachmentEl, attachmentHref);
   } else {
     return renderDefaultPageAttachmentUI(pageEl, attachmentEl);
   }
```

Comment:
This could be simplified by doing:
```js
if (...experimentOn(win)) {
  if (attachmentHref) {
    return renderOutlink();
  } else if (openImgAttr) {
    return renderWithImage();
  }
}
return renderDefault();
```